addons/rose/addon/components/rose/code-editor/toolbar/index.hbs

```diff
@@ -0,0 +1,26 @@
+<div class='code-editor-toolbar' data-test-code-editor-toolbar>
+  <div data-test-code-editor-toolbar-action class='code-editor-toolbar__action'>
+    {{yield to='action'}}
+  </div>
+
+  <div class='code-editor-toolbar__right-aligned'>
+    {{#if (has-block 'menu')}}
+      <div data-test-code-editor-toolbar-menu class='code-editor-toolbar__menu'>
+        {{yield to='menu'}}
+      </div>
+      <div
+        class='code-editor-toolbar__menu-divider'
+        data-test-code-editor-toolbar-menu-divider
+      ></div>
+    {{/if}}
+    <CopyButton
+      @clipboardText={{@copyText}}
+      @success={{this.copied}}
+      class='code-editor-toolbar__copy-button'
+      data-test-code-editor-toolbar-copy-button
+    >
+      <Rose::Icon @name={{this.copyIconType}} @size='medium' />
+      Copy code
```

Comment:
I hardcoded this for now as I was a little unsure of how to proceed.

It looks like in `Rose`, we don't have any translations, I assume because they should be generic and the consumer would be usually supplying any text.

Should this component instead have been in `core`? It's still a little fuzzy for me when it should go to which project. Or should I just pass in another variable that sets the text for the copy button and let the consumer handle it?

Reply:
I think you brought very interesting points.

- Regarding where a component should leave, perhaps @randallmorey can help us? How I always see it is any shared component between UI's goes in Rose, but I also see we have some components within code, so a bit confused.
- About the hard-coded text I am completely fine with it. My humble opinion: let's take simplicity path, for now we just have the use case that require the button with such text. I will think about making that text a parameter of the component when we need it :)

Reply:
Rose components are as simple as they can be and never include human-readable text of any kind.  Also, hard-coded values are generally not used in Boundary UI.  We may want to port this to core.

Reply:
I can port this to core but thought that having it in storybook was really useful, any thoughts about that?

Reply:
Yeah probably having it in storybook is more important for now.  Hopefully this is temporary and we eventually get an official code component.